src/manifestParser/manifestV2.ts

```diff
@@ -74,6 +78,23 @@ export default class ManifestV2 extends ManifestParser<Manifest> {
     return result;
   }
 
+  protected parseInputWebAccessibleScripts(
+    result: ParseResult<Manifest>
+  ): ParseResult<Manifest> {
+    result.manifest.web_accessible_resources?.forEach((resource) => {
+      if (resource.includes("*")) return;
+
+      const inputFile = getInputFileName(resource, this.viteConfig.root);
+      const outputFile = getOutputFileName(resource);
+
+      if (this.pluginExtras.webAccessibleScriptsFilter(inputFile)) {
```

Comment:
What's the reason for checking `inputFile` here instead of checking against `resource` in the check on line 85?

Reply:
Because `inputFile` is resolved to the correct, absolute path of the file, whereas `resource` is some relative path. Each has a different effect on [`createFilter`'s functions](https://github.com/rollup/plugins/blob/master/packages/pluginutils/src/createFilter.ts), which attempt to resolve relative paths to absolute paths on its own using `process.cwd()`. If `resource` was passed, it would resolve to a fake, non-existent absolute path which would not align with true absolute path resolved by `createFilter`.